docs/build.md

```diff
@@ -0,0 +1,118 @@
+# Builds
+
+A Build is a resource object to schedule and run a single [Cloud Native Buildpacks](http://buildpacks.io) build.
+
+Builds provide more granular control than the [Image resource](image.md). However, you will need to manually specify the source, and the cache volume. Additionally, you will need to create a new build for every build execution.
```

Comment:
Does it help to say something like 'Unlike Image, Build doesn't trigger a rebuild on source/builder changes'?

Reply:
I agree.

Reply:
Updated the language to mention the lack of rebuidls. Curious if you have thoughts. 

Also, added a note that builds are immutable.